portability-core/src/main/java/org/dataportabilityproject/shared/ServiceMode.java

```diff
@@ -0,0 +1,9 @@
+package org.dataportabilityproject.shared;
+
+/**
+ * The possible service modes for each authorization
```

Comment:
How about the possible service modes for transfering data. 

In case we use for more than auth later?